.travis.yml

```
@@ -4,6 +4,7 @@ python:
   - "3.6"
 os:
   - linux
+dist: bionic # 18.04
```

```suggestion
dist: 
  - bionic # 18.04
  - focal  # 20.04 LTS
```

I _think_ this is possible?

---

Tried it and it just ran on bionic :/ https://travis-ci.com/github/improbable-eng/perforce-buildkite-plugin/builds/182578913

I think there is a way, may need to write something like:
```
jobs:
  - os: linux
    dist: bionic
  - os:linux
    dist: focal
```

Just gonna leave it for now - mainly just wanted to run on the same version we are actually using in prod

---

Ah, ok.

We're using 16.04 LTS in prod currently.

---

Oh so we are.. oops! Not sure why I had 18.04 in my head..

---

Rampant unbridled lust, I expect ;)